app/dashboard/templates/profiles/profile.html

```diff
@@ -26,12 +26,30 @@
       {% endif %}
     </div>
     <div class="container-fluid bg-light pb-5">
-      {% if not hidden %}
+      </div>
         <div class="container profile-card">
           <div class="row">
             <div class="col-12 col-lg-6 profile-header__main-infos">
+              {% if not hidden %}
+              <div id="float_avatar_image" class="{% if is_my_profile %}is_my_profile{% endif %}">
```

Comment:
Avoid id for styling proposes as much as you can, in that way the code is more modulable, reusable and maintainable. 
also the stylesheet get more comprensible without having to use hierarchy classes  to be able to style when reused or hover states.